sdk/webpubsub/azure-webpubsub-client/azure/webpubsub/client/models/_enums.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum
+from azure.core import CaseInsensitiveEnumMeta
+
+
+class WebPubSubClientState(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    STOPPED = "Stopped"
+    DISCONNECTED = "Disconnected"
+    CONNECTING = "Connecting"
+    CONNECTED = "Connected"
+    RECOVERING = "Recovering"
+
+
+class UpstreamMessageType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    JOIN_GROUP = "joinGroup"
+    LEAVE_GROUP = "leaveGroup"
+    SEND_TO_GROUP = "sendToGroup"
+    SEND_EVENT = "sendEvent"
+    SEQUENCE_ACK = "sequenceAck"
+
+
+class WebPubSubDataType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    BINARY = "binary"
+    JSON = "json"
+    TEXT = "text"
+    PROTOBUF = "protobuf"
+
+
+class CallBackType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
Can we just use a Literal union rather than have an enum for this? If not `Callback` is one word, so no capital "B".

Reply:
callbacktype is very like client state so I make an enum to make it clear. I think it is fine to rename it to `CallbackType`

Reply:
It's fine for preview. We can re-evaluate later.